### ISSUE TITLE
When installing v5.0.0-rc.0: Can't add "solhint-plugin-openzeppelin": invalid package version ""

### ISSUE THREAD

**Issue by SKYBITDev3** [OpenZeppelin/openzeppelin-contracts#4622]
I ran
```
yarn add https://github.com/OpenZeppelin/openzeppelin-contracts#b55b47d42165936876a26279581181757ca9505f
```
and got this error:
> error Can't add "solhint-plugin-openzeppelin": invalid package version "".

**Comment by frangio** [comment]
This is not the way to install the package. Please use `yarn add @openzeppelin/contracts@next` or `yarn add @openzeppelin/contracts@5.0.0-rc.0`.

**Comment by frangio** [comment]
> error Can't add "solhint-plugin-openzeppelin": invalid package version "".

This issue should be fixed by https://github.com/OpenZeppelin/openzeppelin-contracts/commit/ce7e6042a8cb522ef6598778fe4b9483c8bc6d00 but I strongly recommend installing from the npm registry.

**Comment by SKYBITDev3** [comment]
OK thanks, I just thought it wasn't available in npm yet, as I saw 4.9.3 under "Version" at https://www.npmjs.com/package/@openzeppelin/contracts.
